validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkFeedbackRequestValidatorTest.java

```diff
@@ -124,47 +130,40 @@ public class BulkFeedbackRequestValidatorTest extends AbstractValidatorUnitTest
     /**
      * Setup of the Validator and Domain class instance.
      */
-    public void setUpLocalTests() {
+    @BeforeEach
+    @Override
+    public void setUp() {

```

Comment:
Should this be changed back to setUpLocalTests()?  The AbstractSdtUnitTestBase class setUp() method invokes the overridden setupLocalTests() so it would be called by that.  This mechanism allows any set up common to all tests to be added to setUp() and test specific set up in setUpLocalTests().  If it is renamed the @BeforeEach annotation can be removed as this is already on the base class setUp() method.